docs/guides/setup-the-environment.md

```diff
@@ -1,6 +1,6 @@
 # How to setup local environment
 
-This document guides you through the installation and basic configuration of components **needed to create own capability**, so that you can publish it, or use it locally in your application.
+This document guides you through the installation and basic configuration of components **needed to create own API integration**, so that you can publish it, or use it locally in your application.
```

Comment:
I would drop the bold here, but that's maybe outside of scope now.

Reply:
I removed it.